tests/epyccel/test_generic_functions.py

```diff
@@ -6,68 +6,71 @@
 import modules.generic_functions_2 as mod2
 from pyccel.epyccel import epyccel
 
+RTOL = 2e-14
+ATOL = 1e-15
+
```

Comment:
This module does not use any random number. If all tests pass now, I don't see the need to change all exact comparisons to approximate ones.

CC: @EmilyBourne